src/main/java/mate/jdbc/sevice/DriverService.java

```diff
@@ -0,0 +1,16 @@
+package mate.jdbc.sevice;
+
+import java.util.List;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {

```

Comment:
```suggestion
public interface DriverService extends Service<Driver> {
```
🙂

Reply:
ok. But why the app works without this?!